prototypes/multi-module-prototype/examples/multi-module-example/ModulesPrototype/Program.cs

```diff
@@ -70,22 +72,45 @@ public static async Task Main(string[] args)
         var loader = factory.Create(catalogue);
         var moduleCounter = new AsyncCountdownEvent(0);
 
-        var processExplorer = new HostBuilder()
-            .ConfigureLogging(l => l.AddConsole().SetMinimumLevel(LogLevel.Debug))
-            .ConfigureServices(
-             (context, services) => services
-                 .ConfigureSubsystemLauncher(loader.RequestStartProcess, loader.RequestStopProcess, CreateLaunchRequest, CreateStopRequest)
-                 .AddProcessExplorerWindowsServerWithGrpc(pe => pe.UseGrpc())
-                 .Configure<ProcessExplorerServerOptions>(op =>
-                 {
-                     op.Port = 5056;
-                     op.MainProcessId = Process.GetCurrentProcess().Id;
-                     op.EnableProcessExplorer = true;
-                 }))
-            .Build();
-
-        await processExplorer.StartAsync(cts.Token);
-        var infoAggregator = processExplorer.Services.GetRequiredService<IProcessInfoAggregator>();
+        //var processExplorer = new HostBuilder()
+        //    .ConfigureLogging(l => l.AddConsole().SetMinimumLevel(LogLevel.Debug))
+        //    .ConfigureServices(
+        //     (context, services) => services
+        //         .ConfigureSubsystemLauncher(loader.RequestStartProcess, loader.RequestStopProcess, CreateLaunchRequest, CreateStopRequest)
+        //         .AddProcessExplorerWindowsServerWithGrpc(pe => pe.UseGrpc())
+        //         .Configure<ProcessExplorerServerOptions>(op =>
+        //         {
+        //             op.Port = 5056;
+        //             op.MainProcessId = Process.GetCurrentProcess().Id;
+        //             op.EnableProcessExplorer = true;
+        //         }))
+        //    .Build();
+
+        var processExplorer = WebApplication.CreateBuilder(args);
+        processExplorer.Services.AddGrpc();
+        processExplorer.Services.AddCors(o => o.AddPolicy("AllowAll", builder =>
+        {
+            builder.AllowAnyOrigin()
+                    .AllowAnyMethod()
+                    .AllowAnyHeader()
+                    .WithExposedHeaders("Grpc-Status", "Grpc-Message", "Grpc-Encoding", "Grpc-Accept-Encoding");
+        }));
+        processExplorer.Services.ConfigureSubsystemLauncher(loader.RequestStartProcess, loader.RequestStopProcess, CreateLaunchRequest, CreateStopRequest);
+        processExplorer.Services.AddProcessExplorerWindowsServerWithGrpc(pe => pe.UseGrpc());
+        processExplorer.Services.Configure<ProcessExplorerServerOptions>(op =>
+                {
+                    op.Port = 5056;
```

Comment:
Why the 2 port numbers -which used, are different? (5000 in ts and 5056 in C#)

Reply:
5056 wasn't connecting to the server, but then I see listening on 5000 or 5001 in the server console.

Reply:
Something is probably misconfigured. but sure it's okay now as the PR is focusing on the frontend.